src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -333,7 +301,8 @@ TEST_CASE("request_rate_reset_workers: Test the public function ResetWorkers()")
   bool use_mock_infer = true;
   TestRequestRateManager trrm(
       params, is_sequence, is_decoupled, use_mock_infer);
-  trrm.TestResetWorkers();
+  // FIXME! Need to support mocking out the worker
```

Comment:
If this test is disabled, could you comment out the entire `TEST_CASE` block?

Reply:
Also could you explain why this test doesn't work anymore?

Reply:
There was the ability to mock out the 'infer' function before. This test utilized that. Now that does nothing because the infer function isn't in this class anymore (and the test hangs).  I will fix and enable.